src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java

```diff
@@ -94,7 +94,7 @@
 	 */
 	public static void main(String[] args) {
 		try {
-			PropertyConfigurator.configure(new File("./rsc/log4j.properties").getAbsolutePath()); //$NON-NLS-1$
+			PropertyConfigurator.configure(new File("./src/main/resources/log4j.properties").getAbsolutePath()); //$NON-NLS-1$
```

Comment:
you can try to read it from classpath instead of being dependent on project structure, sth like this: https://howtodoinjava.com/java/io/read-file-from-resources-folder/

Reply:
This is a main method for testing purposes, so no need to focus to much on it, but in general we have the same issue in other parts of the code, we should create an issue to fix them.